src/DependencyInjection/Configuration.php

```diff
@@ -24,8 +24,14 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('mvo_contao_facebook_import');
+        $treeBuilder = new TreeBuilder('mvo_contao_facebook_import');
+
+        // Keep compatibility with symfony/config < 4.2
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            $rootNode = $treeBuilder->getRootNode();
+        } else {
+            $rootNode = $treeBuilder->root('mvo_contao_facebook_import');
+        }
```

Comment:
Since Contao 4.4 reached the end of its bugfix live, it might be easier to just require `symfony/config: ^4.4 || ^5.2`.

Reply:
You're right, it's kind of philosophy how long @m-vo wants to support Contao 4.4.
I just copied the logic from other extensions which still support Contao 4.4 and Symfony 5 ;-)

Reply:
push @m-vo 😁

Reply:
Ah sorry, this went off my radar for some reason.

Yes, let's please go with Sf `^4.4 || ^5.2` and also contao/core-bundle `^4.9` as lower boundary. May I ask you to please also drop this nasty bit in this go? https://github.com/m-vo/contao-facebook-import/blob/9b86a40a10c593954007fa0d56a55af719620cc5/src/Resources/contao/config/config.php#L38-L49 :smile:

Reply:
Done!